data/mods/Aftershock/maps/terrain_groundxeno.json

```diff
@@ -114,5 +96,309 @@
+    "bash": {
+      "str_min": 65,
+      "str_max": 160,
+      "sound": "smash!",
+      "sound_fail": "thump.",
+      "items": [ { "item": "rock", "count": [ 3, 4 ] }, { "item": "sharp_rock", "count": [ 1, 2 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_snow",
+    "name": "snow",
+    "looks_like": "t_dirt_season_winter",
+    "description": "A patch of thick snow, very cold.",
+    "symbol": ".",
+    "color": "light_blue",
+    "move_cost": 4,
+    "flags": [ "TRANSPARENT", "DIGGABLE", "FLAT" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 100,
+      "sound": "splosh!",
+      "sound_fail": "fweet!",
+      "ter_set": "t_deaddirt",
+      "str_min_supported": 100
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_snowrock",
+    "name": "snow",
+    "looks_like": "t_dirt_season_winter",
+    "description": "A patch of thick snow on top of rocks, very cold.",
+    "symbol": ".",
+    "color": "light_blue",
+    "move_cost": 4,
+    "flags": [ "TRANSPARENT", "DIGGABLE", "FLAT" ],
+    "bash": {
+      "str_min": 8,
+      "str_max": 100,
+      "sound": "splosh!",
+      "sound_fail": "fweet!",
+      "ter_set": "t_basaltfloor",
+      "str_min_supported": 100
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_snow_roof",
+    "name": "snow",
+    "looks_like": "t_dirt_season_winter",
+    "description": "A patch of thick snow, very cold.",
+    "symbol": ".",
+    "color": "light_blue",
+    "move_cost": 4,
+    "flags": [ "TRANSPARENT", "DIGGABLE", "FLAT" ],
+    "bash": {
+      "str_min": 80,
+      "str_max": 100,
+      "sound": "splosh!",
+      "sound_fail": "fweet!",
+      "ter_set": "t_null",
+      "str_min_supported": 100,
+      "bash_below": true
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_basalt_wall",
+    "name": "basalt wall",
+    "looks_like": "t_rock",
+    "description": "A sturdy basalt wall.",
+    "symbol": "LINE_OXOX",
+    "color": "light_gray",
+    "move_cost": 0,
+    "coverage": 100,
+    "roof": "t_basaltfloor",
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": {
+      "str_min": 80,
+      "str_max": 300,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "ter_set": "t_null",
+      "items": [ { "item": "rock", "count": [ 8, 18 ] } ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_ice_wall",
+    "name": "glacial ice wall",
+    "description": "A solid wall of glacial ice.",
+    "symbol": "LINE_OXOX",
+    "color": "light_blue",
+    "move_cost": 0,
+    "coverage": 100,
+    "roof": "t_ice",
+    "flags": [ "NOITEM", "SUPPORTS_ROOF", "WALL", "NO_SCENT", "AUTO_WALL_SYMBOL", "MINEABLE", "BLOCK_WIND" ],
+    "connects_to": "WALL",
+    "bash": { "str_min": 50, "str_max": 100, "sound": "crash!", "sound_fail": "whump!", "ter_set": "t_null" }
```

Comment:
Why bash below for this? Its a wall so you aren't exactly bashing below like with the ice floor

Reply:
Mostly because you are making it turn into `t_null`, which creates open air. If you change the `ter_set` into an ice floor then you can omit bash below.

Reply:
Oh, I'm a doofus I figured it would just delete, let me fix that and turn it into an ice floor